.travis.yml

```diff
@@ -0,0 +1,42 @@
+sudo: false
+
+language: php
```

Comment:
important notice: outsources the maintenance of the PHP environments to Travis CI for the library to be cross-checked across PHP versions